lib/create-contentful-api.js

```diff
@@ -153,6 +157,71 @@ export default function createClientApi ({ http }) {
       .then((response) => wrapOrganizationCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets a collection of usage periods for the organization. Only available to Admin or owner of the organization.
+   * @memberof ContentfulClientAPI
+   * @param organizationId
+   * @return {Promise<UsagePeriodCollection>} Promise for a collection of usage periods
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * const client = contentful.createClient({
+   *   accessToken: '<content_management_api_key>'
+   * })
+   *
+   * client.getUsagePeriods(organizationId)
+   * .then(result => console.log(result.items))
+   * .catch(console.error)
+   */
+  function getUsagePeriods (organizationId) {
+    const baseURL = http.defaults.baseURL.replace('/spaces/', `/organizations/${organizationId}/usage_periods`)
+    const headers = {
+      'x-contentful-enable-alpha-feature': 'usage-insights'
+    }
+    return http.get('', {
+      baseURL,
+      headers
+    })
+      .then((response) => wrapUsagePeriodCollection(http, response.data), errorHandler)
+  }
+
+  /**
+   * Gets a collection of usages for the organization. Only available to Admin or owner of the organization.
```

Comment:
In general, I wonder if I'm giving too much context in these doc strings?

Reply:
Looks good enough having the code example will help the user getting more context